lib/internal/test_runner/reporter/spec.js

```diff
@@ -57,15 +59,29 @@ class SpecReporter extends Transform {
       RegExpPrototypeSymbolSplit(
         hardenRegExp(/\r?\n/),
         inspectWithNoCustomRetry(err, inspectOptions),
-      ), `\n${indent}  `);
-    return `\n${indent}  ${message}\n`;
+      ), indent !== undefined ? `\n${indent}  ` : '');
+    return `${indent !== undefined ? `\n${indent}  ` : '\n'}${message}\n`;
   }
-  #handleEvent({ type, data }) {
+  #formatTestReport(type, data, prefix = '', indent = undefined, hasChildren = false, skippedSubtest = false, showFilePath = false) {
     let color = colors[type] ?? white;
     let symbol = symbols[type] ?? ' ';
-
+    const duration_ms = data.details?.duration_ms ? ` ${gray}(${data.details.duration_ms}ms)${white}` : '';
+    const title = `${showFilePath ? `${data.file} ` : ''}${data.name}${duration_ms}${skippedSubtest ? ' # SKIP' : ''}`;
```

Comment:
```suggestion
    const title = `${data.name}${duration_ms}${skippedSubtest ? ' # SKIP' : ''}`;
```
seems like an unrelated change, let's revert it.

Reply:
Initially I printed the failed test reports in different format.

After thinking for a while, I think let's stick with being consistent. :smile:
I undo-ed the different format changes.